ui/components/app/wallet-overview/coin-buttons.tsx

```diff
@@ -116,6 +121,13 @@ type CoinButtonsProps = {
   iconButtonClassName?: string;
 };
 
+///: BEGIN:ONLY_INCLUDE_IF(build-flask)
+const multichainAccountTypeToSnapId = {
+  [BtcAccountType.P2wpkh]: BITCOIN_WALLET_SNAP_ID,
+  [SolAccountType.DataAccount]: SOLANA_WALLET_SNAP_ID,
+};
+///: END:ONLY_INCLUDE_IF
```

Comment:
The Solana support should be code-fenced with the `solana` feature here no?

IIRC we cannot nest code-fencing, so that's not going to look pretty, but this should work (assuming `build-flask` is always enabled if `solana` is enabled too):

```suggestion
///: BEGIN:ONLY_INCLUDE_IF(build-flask)
const multichainAccountTypeToSnapId = {
  [BtcAccountType.P2wpkh]: BITCOIN_WALLET_SNAP_ID,
///: END:ONLY_INCLUDE_IF
///: BEGIN:ONLY_INCLUDE_IF(solana)
  [SolAccountType.DataAccount]: SOLANA_WALLET_SNAP_ID,
///: END:ONLY_INCLUDE_IF
///: BEGIN:ONLY_INCLUDE_IF(build-flask)
};
///: END:ONLY_INCLUDE_IF
```

But I was also thinking of this other solution otherwise:

```suggestion
// This variable might be unused if we are not building for flask. Since code-fence cannot
// be nested, we choose to ignore this eslint warning rather than having multiple code-fences.
// eslint-disable-next-line no-unused-vars
const multichainAccountTypeToSnapId = {
  ///: BEGIN:ONLY_INCLUDE_IF(build-flask)
  [BtcAccountType.P2wpkh]: BITCOIN_WALLET_SNAP_ID,
  ///: END:ONLY_INCLUDE_IF
  ///: BEGIN:ONLY_INCLUDE_IF(solana)
  [SolAccountType.DataAccount]: SOLANA_WALLET_SNAP_ID,
  ///: END:ONLY_INCLUDE_IF
};
```

Reply:
Also, this should be probably named `MULTICHAIN_ACCOUNT_TYPE_TO_SNAP_ID`. We could also move this to `shared/constants/multichain/accounts.ts` maybe?